apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/observe/api_collections/ApiDetails.jsx

```diff
@@ -30,6 +30,20 @@ function ApiDetails(props) {
     const [showMoreActions, setShowMoreActions] = useState(false)
     const setSelectedSampleApi = PersistStore(state => state.setSelectedSampleApi)
 
+    const statusFunc = getStatus ? getStatus : (x) => {
+        try {
+            if (paramList && paramList.length > 0 &&
+                paramList.filter(x => x?.nonSensitiveDataType).map(x => x.subTypeString).includes(x)) {
+                return "info"
+            }
+        } catch (e) {
+
+        }
+        return "warning"
+    }
+
+    const skipList = ["GENERIC", "TRUE", "FALSE","INTEGER_32", "INTEGER_64", "NULL", "OTHER", "DICT", "FLOAT"]
```

Comment:
make this set and in some func or transform file where if needed can be used in some more places in dashboard